Deploy-CAM.ps1

```diff
@@ -2694,10 +2688,17 @@ else {
             }
         }
 
-		
-        if ($domainAdminCredential.GetNetworkCredential().Password.Length -lt 12) {
+        if ((-not ($domainAdminCredential.UserName -imatch '\w+')) -or ($domainAdminCredential.Username.Length -gt 20)) {
+            Write-Host "Please enter a valid username. It can only contain letters and numbers and cannot be longer than 20 characters."
+            $domainAdminCredential = $null
+            continue
+        }
+
+        if ((-not $deployOverDC) `
+            -and ( $domainAdminCredential.GetNetworkCredential().Password.Length -lt 12 ) ) {
```

Comment:
Should we add in the other checks here as well? @bdyck ran into that issue this week where he had "admin" in the Password or something and it errored out much later

Reply:
Good point - let's make a ticket for those. I think that was to do with the username and not the password, but I think we should also add the 'username should not be in password' check and maybe a couple other things. The reason this is in this branch is because this fix isn't a 'try it again' type thing - the service account would need to be modified if their service account doesn't match our password policy, which is silly and inconvenient. Luckily one of Jeremy's accounts met our threshold.